docs/self-managed/backup-restore/optimize-backup.md

```diff
@@ -0,0 +1,127 @@
+backup:
+  repositoryName: <repository name>
+```
+
+## Create backup API
+
+Please note that the backup API can be reached via the `/actuator` management port, by default this is `8092`.  
+The following endpoint can be used to asynchronously trigger the backup process:
+
+```
+POST actuator/backup
+{
+  "backupId": <backupId>
+}
+```
+
+### Response
+
+| Code             | Description                                                                                                                                                                                                                                 |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 200 OK           | Backup was successfully started, snapshots will be created asynchronously. List of snapshot names is returned in response body (see example below). This list must be persisted together with the backup ID to be able to restore it later. |
+| 400 Bad Request  | Indicates issues with the request, for example when a backup with the same `backupId` already exists.                                                                                                                                       |
+| 500 Server Error | All other errors, e.g. issues communicating with Elasticsearch for snapshot creation.                                                                                                                                                       |
+
+### Example request
+
+```
+curl --request POST 'http://localhost:8092/actuator/backup' \
+-H 'Content-Type: application/json' \
+-d '{ "backupId": "backup1" }'
+```
+
+### Example response
+
+```json
+{
+  "scheduledSnapshots": [
+    "camunda_optimize_backup1_3.9.0_part_1_of_2",
+    "camunda_optimize_backup1_3.9.0_part_2_of_2"
+  ]
+}
+```
+
+## Check backup state API
+
+Please note that the backup API can be reached via the `/actuator` management port, by default this is `8092`.  
+As the backup is created asynchronously, the current state of the backup can be checked by calling the following endpoint:
+
+```
+GET actuator/backup/{backupId}
+```
+
+### Response
+
+| Code             | Description                                                                                 |
+| ---------------- | ------------------------------------------------------------------------------------------- |
+| 200 OK           | Backup state could be determined and is returned in the response body (see example below)   |
+| 404 Not Found    | Backup with given id does not exist.                                                        |
+| 500 Server Error | All other errors, e.g. issues communicating with Elasticsearch for snapshot state retrieval |
+
+### Example request
+
+```
+curl 'http://localhost:8092/actuator/backup/backup1'
```

Comment:
we should include the method in this request

Reply:
Can you also add this comment to the other PRs please so they can adjust the same